Installation/include/CGAL/config.h

```diff
@@ -152,6 +152,10 @@
     defined(BOOST_NO_CXX11_HDR_ARRAY) || BOOST_VERSION < 104000
 #define CGAL_CFG_NO_CPP0X_ARRAY 1
 #endif
+#if defined(BOOST_NO_CXX11_HDR_UNORDERED_SET) || \
+    defined(BOOST_NO_CXX11_HDR_UNORDERED_MAP)
+#define CGAL_CFG_NO_CPP0X_UNORDERED 1
+#endif
```

Comment:
Be careful that in boost 1.48 the macro seems to be named `BOOST_NO_0X_HDR_UNORDERED_SET`